loadgen/README.md

```diff
@@ -103,3 +99,67 @@ with the reference models.
 
 For templates of how to do the above in detail, refer to code for the demos,
 tests, and reference models.
+
+
+## Submissions over the network
+
+For reference, on a high level a submission looks like this:
+
+<div align="center" style="display:flex; flex-flow:row wrap; justify-content: space-evenly;">
+<img src="diagram_submission.png" width="300px" style="padding: 20px">
+</div>
+
+The loadgen implementation is common to all submissions, while the QSL (“Query Sample Library”) and SUT (“System Under Test”) are implemented by submitters. QSL is responsible for loading the data and includes untimed preprocessing.
+
+A submission over the network introduces a new component “QDL” (query dispatch library) and a it would look like this:
+
+<div align="center" style="display:flex; flex-flow:row wrap; justify-content: space-evenly;">
+<img src="diagram_network_submission.png" width="300px" style="padding: 20px">
+</div>
+
+QDL is a proxy for a load-balancer, that dispatches queries to SUT over a physical network, receives the responses and passes them back to loadgen.  It is implemented by the submitter. The interface of the QDL is the same as the API to SUT. 
+
+In scenarios using QDL, data in QSL may be compressed at the choice of the submitter in order to reduce network transmission time. Decompression is part of the timed processing in SUT. A set of approved standard compression schemes will be specified for each benchmark; additional compression schemes must be approved in advance by the Working Group.
```

Comment:
I suggest "data may be compressed in QSL" - to clarify that QSL is not just the owner of the data, but also where compression happens.

Reply:
Agree. I will make the change